addons/udes_stock/models/stock_move.py

```diff
@@ -175,3 +175,98 @@ def split_out_move_lines(self, move_lines, **kwargs):
                 (new_move | self).update_orig_ids(self.move_orig_ids)
 
         return new_move
+
+    def push_from_drop(self):
+        """
+        Creates new moves for the moves that have just dropped stock in a location.  
+        """
+        Move = self.env["stock.move"]
+        MoveLine = self.env["stock.move.line"]
+        Push = self.env["stock.rule"]
+
+        done_moves = self.filtered(lambda m: m.state == "done")
+
+        # load all the move lines, grouped by location
+        move_lines_by_location = done_moves.mapped("move_line_ids").groupby("location_dest_id")
```

Comment:
Do we still need the mapped?

Reply:
@jrbirch99 To expand, instead of .mapped("x") you can write .x in most cases.